src/modules/timer.rs

```diff
@@ -0,0 +1,83 @@
+use crate::config::TableExt;
```

Comment:
In #116, I unify `crate::config::TableExt` and `crate::config::Config`.
Maybe we should try merging that in first to avoid creating conflicts.